LoadTests/SET_UP_TEST_ENTITIES.sql

```diff
@@ -1,126 +1,93 @@
+
+                INSERT INTO "OrganisationSicCodes"
+                ("OrganisationSicCodeId", "OrganisationId", "SicCodeId", "Created", "Source")
+                VALUES
+                (STARTING_ID + INDEX, STARTING_ID + INDEX, 2200, '01/10/2019 12:26:44', 'User');
+
+                INSERT INTO "UserOrganisations"
+                ("UserId", "OrganisationId", "ConfirmAttempts", "Created", "Modified", "PINConfirmedDate")
+                VALUES
+                (STARTING_ID + INDEX, STARTING_ID + INDEX, 0, '01/10/2019 12:26:44', '01/10/2019 12:26:44', '01/10/2019 12:26:44');
+
+                -- Create test organisations that are NOT linked to test users
+                INSERT INTO "Organisations"
+                ("OrganisationId", "CompanyNumber", "OrganisationName", "SectorTypeId", "StatusId", "StatusDate", "StatusDetails", "Created", "Modified", "OptedOutFromCompaniesHouseUpdate", "EmployerReference")
+                VALUES
+                (STARTING_ID + NUM_OF_USERS + INDEX, '99999' || CAST(INDEX + NUM_OF_USERS AS VARCHAR(16)), 'test_' || CAST(INDEX + NUM_OF_USERS  AS VARCHAR(16)), 1, 3, '01/10/2019 12:26:44', 'PIN Confirmed', '01/10/2019 12:26:44', '01/10/2019 12:26:44', true, 'ABCDE' || CAST(INDEX + NUM_OF_USERS AS VARCHAR(16)));
+
+                INSERT INTO "OrganisationNames"
+                ("OrganisationNameId", "OrganisationId", "Name", "Source", "Created")
+                VALUES
+                (STARTING_ID + NUM_OF_USERS + INDEX, STARTING_ID + NUM_OF_USERS + INDEX, 'test_' || CAST(INDEX + NUM_OF_USERS AS VARCHAR(16)), 'User', '01/10/2019 12:26:44');
+
+                INSERT INTO "OrganisationScopes"
+                ("OrganisationScopeId", "OrganisationId", "ScopeStatusId", "ScopeStatusDate", "RegisterStatusId", "RegisterStatusDate", "SnapshotDate", "StatusId")
+                VALUES
+                (STARTING_ID + NUM_OF_USERS + INDEX, STARTING_ID + NUM_OF_USERS + INDEX, 1, '01/10/2019 12:26:44', 0, '01/10/2019 12:26:44', '05/04/2017 00:00:00', 3);
+
+                INSERT INTO "OrganisationAddresses"
+                ("AddressId", "OrganisationId", "CreatedByUserId", "Address1", "Country", "PostCode", "StatusId", "StatusDate", "Created", "Source", "IsUkAddress")
+                VALUES
+                (STARTING_ID + NUM_OF_USERS + INDEX, STARTING_ID + NUM_OF_USERS + INDEX, STARTING_ID + NUM_OF_USERS + INDEX, 'test street', 'United Kingdom', 'NWN NWN', 3, '01/10/2019 12:26:44', '01/10/2019 12:26:44', 'User', 'True');
+
+                INSERT INTO "OrganisationSicCodes"
+                ("OrganisationSicCodeId", "OrganisationId", "SicCodeId", "Created", "Source")
+                VALUES
+                (STARTING_ID + NUM_OF_USERS + INDEX, STARTING_ID + NUM_OF_USERS + INDEX, 2200, '01/10/2019 12:26:44', 'User');
+
+                INDEX := INDEX + 1;
+            END LOOP;
+
+        perform * from "Organisations" where "Organisations"."OrganisationId" >= STARTING_ID;

```

Comment:
Neat :D Should this be capitalised?

Reply:
It doesn't matter, but probably - for some reason IDEA and DataGrip change the capitalisation of my files weirdly - this was capitalised when I ran it!